### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by filipblnt**
see https://github.com/blindnet-io/devrel-management/issues/9 & https://github.com/blindnet-io/openness-framework/pull/33

### CODE REVIEW

**Comment by nweldev** [comment]
"server-side" and "node.js" are (close to being) redundant

In most cases, I don't think we need this repetition.
IMHO, this repo name should reflect the name of the npm package (`sdk-node`).

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+  <a href="https://github.com/blindnet-io/{project-short-name}/issues">Submit an Issue</a>
+  &nbsp;•&nbsp;
+  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
+  <br>
+  <br>
+</p>
+
+## About
+
+This is the JavaScript [server-side SDK](https://docs.blindnet.io/docs/extra/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```
Comment (m4rk055) [inline README.md]:
```suggestion
This is the JavaScript [server-side SDK](https://blindnet.dev/docs/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+
+- Generate [authentication tokens](https://docs.blindnet.io/docs/extra/glossary#authentication-token) to allow users to access blindnet.
```
Comment (m4rk055) [inline README.md]:
```suggestion
- Generate [authentication tokens](https://blindnet.dev/docs/glossary#authentication-token) to allow users to access blindnet.
```

**Review by m4rk055 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+## Get Started
+
+:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
+
+## Installation
+
+Use [npm][npm] or [yarn][yarn] to install the `@blindnet/sdk-javascript` package:
+
+```bash
+npm install @blindnet/sdk-node
```
Comment (m4rk055) [inline README.md]:
```suggestion
npm install -S @blindnet/sdk-node
```

**Pull request merged**